itou/templates/static/legal/terms.html

```diff
@@ -22,274 +18,754 @@ <h2 class="h3">
         <div class="s-section__container container">
             <div class="row">
                 <div class="col-12">
-                    <h2 id="article-1---champ-dapplication">Article 1 - Champ d’application</h2>
+                    <h2>Article 1 - Champ d'application</h2>
                     <p>
-                        Le site des emplois de l&#39;inclusion est d’accès libre et gratuit à tout Utilisateur. La simple visite des emplois de l&#39;inclusion suppose l’acceptation par tout Utilisateur des présentes conditions générales d’utilisation.
+                        Les présentes conditions générales d'utilisation (ci-après dites
+                        « CGU ») précisent le cadre juridique des "Emplois de l'inclusion"
```

Comment:
```suggestion
                        « CGU ») précisent le cadre juridique des "emplois de l'inclusion"
```

De partout en minuscules

Reply:
Je commence à douter : dans les anciennes CGU c'est majoritairement en minusciles, ici en majuscules.

À trancher mais à uniformiser je dirais…